centos7-init.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+# Copyright 2021 Cloudera, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# This script sets up and environment suitable for running Ansible playbooks using Centos 7. It is distinct from the docker version of runner.
```

Comment:
Nitpick: Add a space to separate the license.

We will need entries in the README for this script.

Reply:
Added